services/bitrise/bitrise.service.js

```diff
@@ -24,9 +24,14 @@ module.exports = class Bitrise extends LegacyService {
     return [
       {
         title: 'Bitrise',
-        previewUrl: 'cde737473028420d/master?token=GCIdEzacE4GW32jLVrZb7A',
-        pattern: ':app-id/:branch?token=:app-status-badge-token',
-        exampleUrl: 'cde737473028420d/master?token=GCIdEzacE4GW32jLVrZb7A',
+        pattern: ':appId/:branch',
+        namedParams: { appId: 'cde737473028420d', branch: 'master' },
+        queryParams: { token: 'GCIdEzacE4GW32jLVrZb7A' },
```

Comment:
I noticed in the review app that the path is displaying as:

`/bitrise/:appId/:branch.svg?token=GCIdEzacE4GW32jLVrZb7A` 

With the token query param value plugged in directly. Is that the expected behavior for query params? Is it possible to still have that display as `/bitrise/:appId/:branch.svg?token=:token` ?

Reply:
Yea, it is for now. Not yet: see #2357.